tailwind.config.js

```diff
@@ -0,0 +1,49 @@
+const colors = require("tailwindcss/colors");
+
+module.exports = {
+  content: [
+    "./components/**/*.js",
+    "./components/**/*.tsx",
+    "./nextra-theme-docs/**/*.js",
+    "./nextra-theme-docs/**/*.tsx",
+    "./nextra-theme-docs/**/*.css",
```

Comment:
NEVER include `css` files in content field https://github.com/shuding/nextra/pull/495